.github/workflows/build-test.yaml

```diff
@@ -1,13 +1,13 @@
 name: build-test
 
 on:
-  push:
-    branches:
-      - "**"
+  pull_request_target:
```

Comment:
This line should have a comment linking to the document explaining how this can be unsafe